src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -81,6 +81,9 @@ namespace Microsoft.Terminal.Settings.Model
         INHERITABLE_PROFILE_SETTING(Boolean, SnapOnInput);
         INHERITABLE_PROFILE_SETTING(Boolean, AltGrAliasing);
         INHERITABLE_PROFILE_SETTING(BellStyle, BellStyle);
+
+        INHERITABLE_PROFILE_SETTING(Windows.Foundation.Collections.StringMap, EnvironmentVariables);

```

Comment:
nit on use of `StringMap`; it has observation capabilities and therefore a bit of an insert overhead, and we don't expect the contents to change during runtime

see earlier note about using a view as well. not sure how nicely that plays with the macros...